lib/ansible/modules/packaging/language/pip.py

```diff
@@ -557,7 +590,11 @@ def main():
                                 out += '%s\n' % formatted_dep
 
                 for pkg in name:
-                    is_present = _is_present(pkg, version, pkg_list, pkg_cmd)
+                    pkg_name, version_formula = _parse_name_str(pkg)
```

Comment:
This code basically repeats one above, which supports my previous point that having a separate class for distribution entity would be a good abstraction to have.